src/atomate2/abinit/jobs/base.py

```diff
@@ -136,16 +187,16 @@ def calc_type(self) -> str:
         """Get the type of calculation for this maker."""
         return self.input_set_generator.calc_type
 
-    @job
+    @abinit_job
     def make(
         self,
         structure: Structure | None = None,
         prev_outputs: str | Path | list[str] | None = None,
         restart_from: str | Path | list[str] | None = None,
         history: JobHistory | None = None,
+        stop_jobflow: bool = False,
```

Comment:
In my case it is quite useful because if one perturbation fails, then there is no need to launch the other ones (if they haven't been launched already) so this saves time.
I changed its name and I put it as an attribute of the Maker